src/components/OrgPostCard/OrgPostCard.tsx

```
@@ -365,12 +380,60 @@ export default function OrgPostCard({
               <Card.Title className={styles.titleOrgPostCard}>
-                {post.caption}
+                <div
+                  ref={captionRef}
+                  style={{
+                    lineHeight: '1.6em',
+                    whiteSpace: 'pre-wrap',
+                    wordBreak: 'break-word',
+                    marginBottom: '8px',
+                  }}
+                  aria-expanded={expanded}
+                >
```

Biome (2.1.2)

[error] The ARIA attribute 'aria-expanded' is not supported by this element.

Ensure that ARIA attributes are valid for the role of the element.

(lint/a11y/useAriaPropsSupportedByRole)